src/docs/style/color.mdx

```diff
@@ -22,7 +22,7 @@ Brightlayer UI offers a variety of different colors for use in your applications
 
 Our color sets are divided into sections as outlined below. User Interface and Status colors are available in a single package ([@brightlayer-ui/colors](https://www.npmjs.com/package/@brightlayer-ui/colors)). Branding (Charts and Graphs) colors are available as an additional package ([@brightlayer-ui/colors-branding](https://www.npmjs.com/package/@brightlayer-ui/colors-branding)).
 
-This color pallette depicts the Material Design 2 colors and Material Design 3 colors will be substantially different. React Native projects with Material Design v2 should use version v3 of colors package and React Native projects with Material Design v3 should use version v4 of colors package. We highly recommend to use semantic colors from the themes package([@brightlayer-ui/react-native-themes](https://www.npmjs.com/package/@brightlayer-ui/react-native-themes)) instead of directly using explicit colors from the colors package.
+> This color palette depicts the MD2 colors and MD3 colors will be substantially different. RN projects with MD2 should use version v3 of colors package and RN projects with MD3 should use version v4 of colors package. We highly recommend to use semantic colors from the themes package ([@brightlayer-ui/react-native-themes](https://www.npmjs.com/package/@brightlayer-ui/react-native-themes)) instead of directly using explicit colors from the colors package.
```

Comment:
```suggestion
> This color palette depicts Material Design 2 (MD2) compatible colors — Material Design 3 (MD3) will use a new color system. If you are using @brightlayer-ui/colors, version 3 follows the structure below and version 4 follows the newer MD3 structure. Only React Native projects using React Native Paper version 5 should use the MD3 colors at this time. Other projects should continue to use the MD2 colors below. When using colors, we highly recommend you use semantic colors from our themes whenever possible instead of using colors directly from the colors package.
```